DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Processors/Factories/DicomProcessorFactory.cs

```diff
@@ -22,8 +26,23 @@ public IAnonymizerProcessor CreateProcessor(string method, JObject settingObject
                 "remove" => new RemoveProcessor(),
                 "refreshuid" => new RefreshUIDProcessor(),
                 "keep" => new KeepProcessor(),
-                _ => null,
+                _ => CreateCustomProcessor(method)
             };
         }
+
+        public void AddCustomerProcessor(string method, IAnonymizerProcessor processor)
+        {
+            EnsureArg.IsNotNullOrEmpty(method, nameof(method));
+            EnsureArg.IsNotNull(processor, nameof(processor));
+
+            _customerProcessors[method.ToLower()] = processor;
+        }
+
+        private IAnonymizerProcessor CreateCustomProcessor(string method)
```

Comment:
What's the expected behavior of this function? Instead of `Create`, it looks more like a `Get` operation as the custom processor is created in `AddCustomProcessor`.

Reply:
Yes, I removed this function here.